src/main/webapp/timeline.html

```diff
@@ -7,14 +7,9 @@
     <script src="script.js"></script>
   </head>
   <body>
-     <img src="images/colorBlockStripe.jpg" class="stripe"/>
-     <div class="profCard">
-         <img src="images/profpic.jpg" class="profPicDefault"/>
-         <h1>Name</h1>
-         <h2>Skate Type</h2>
-         <p class=bioText>Bio sample: Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.</p>
-         <a href="editbio.html" class="editButton">Edit</a>
-      </div>
+      <img src="images/logo.png" class="logoTimeline"/>
+      <div class='box'></div>
+      <div class="profInfo"><img src="images/profpic.jpg" class="profPicDefault"/><h1>Name</h1><h2>Skate Type</h2><p class=bioText>Bio sample: Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor.</p><button class="editButton"><a href="/editbio.html">Edit</a></button></div>
```

Comment:
This line is really long. Whenever you have really long lines like this, it's better to break it up int multiple lines. Also, this stuff is nested so at the least it should look like this:
```suggestion
      <div class="profInfo">
        <img src="images/profpic.jpg" class="profPicDefault"/>
        <h1>Name</h1>
        <h2>Skate Type</h2>
        <p class=bioText>Bio sample: Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor.</p>
        <button class="editButton"><a href="/editbio.html">Edit</a></button>
      </div>
```

Actually, that fixes the line length issue